andela/ah-titans
# #159330641 Add base template and templates of merged feature

#### What does this PR do?
Seeks to merge the ch-add-base-template-159330641 branch into the develop branch.

#### Description of tasks to be completed
Create the base mock template and the mock template for the already merged features.

#### How should this be manually tested?
```sh
$ git clone https://github.com/andela/ah-titans.git
```
```sh
$ git checkout ch-add-base-template-159330641
```
```sh
$ cd ah-titans/authors/templates
```

> Then you can double-click on the individual templates to view them on the browser.`
> You should be able to see the pages like the ones shown below.

>Base template

<img width="1280" alt="screen shot 2018-07-31 at 12 35 32" src="https://user-images.githubusercontent.com/27728464/43451763-5e3838a8-94be-11e8-912e-7371dedd5069.png">



>Login template

<img width="1280" alt="screen shot 2018-07-31 at 10 58 49" src="https://user-images.githubusercontent.com/27728464/43446682-2e4c4d0c-94b2-11e8-99f8-8e0df2bb28e7.png">


>Signup template

<img width="1280" alt="screen shot 2018-07-30 at 19 46 10" src="https://user-images.githubusercontent.com/27728464/43440283-6817477a-949f-11e8-8e19-27e73aad093b.png">


#### Any background context you want to provide?
No

#### What are the relevant Pivotal Tracker Stories?
[#159330641](https://www.pivotaltracker.com/story/show/159330641)
